frappe_graphql/utils/generate_sdl/root.graphql

```diff
@@ -0,0 +1,86 @@
+schema {
```

Comment:
What is the motivation behind keeping the root  in the repo versus the developer generating it?

Reply:
Earlier, we had `root.py` which was basically string concatenation to get the same content.
Now, instead, we copy contents of root.py

Reply:
I see. My only problem with this is that the IDE will complain of duplicate types (no errors with running GraphQL) if we have the root file in another app 😬 . @e-lobo What do you think?

Reply:
unlucky 😄

Reply:
yea ide complains of duplicate types. its good that developer has the option to always `ignore_root_file` although now by default generation of `root.graphql` is not needed. so my suggestion would be `ignore_root_file` default to `True` instead of `False` ie by default do not copy the root.graphql to target dir maybe change the flag name too  @assemmarwan @faztp12 what say?

Reply:
but then, there won't be any root file for the GraphQLSchema, right ? this particular root.graphql is not included via hooks 😬

Reply:
OR,
lets have the `root.graphql` defined in frappe_graphql itself
`generate_sdl` will no more generate root.graphql.